content/components/index.mdx

```diff
@@ -33,6 +33,15 @@ import {Grid, Flex, Box, Link, Text, Label} from '@primer/components'
         <Link href="/design/components/tokens" fontWeight="bold">Tokens</Link>
         <Box as="p">A token is a compact representation of an object, and is typically used to show a collection of related metadata.</Box>
     </div>
+    <div>
```

Comment:
This should be after action list to keep a-z order. Needs image and updated description.